.github/workflows/osx.yml

```diff
@@ -10,7 +17,7 @@ on:
         default: false
 
 concurrency:
-  group: ${{ github.workflow}}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ (github.head_ref && github.ref) || github.run_id }}
```

Comment:
what's && here? are they trancated together

Reply:
This is basically "if (head_ref) { github.ref } else { run_id }"

Reply:
okay, it is still in the fallback behavior